internal/controllers/machine/machine_controller_phases.go

```diff
@@ -187,11 +197,6 @@ func (r *Reconciler) reconcileBootstrap(ctx context.Context, cluster *clusterv1.
 		return ctrl.Result{}, nil
 	}
 
-	// Call generic external reconciler if we have an external reference.
-	externalResult, err := r.reconcileExternal(ctx, cluster, m, m.Spec.Bootstrap.ConfigRef)
-	if err != nil {
-		return ctrl.Result{}, err
-	}
 	if externalResult.RequeueAfter > 0 {
```

Comment:
Should we move up this line and the next 6 as well?

It's a bit unintuitive to handle the result so late